src/ray/protobuf/gcs.proto

```diff
@@ -276,6 +276,11 @@ message ResourceLoad {
   repeated ResourceDemand resource_demands = 1;
 }
 
+message PlacementGroupLoad {
+  // The list of pending placement group specifications.
+  repeated PlacementGroupTableData placement_group_data = 1;
```

Comment:
I include the whole table for simplicity. Additional information will be

```
name
placement_group_id
bundle_id
placement group state # This could be useful for autoscaling I guess?
```